docs/ADRs/0007-commit-convention.md

```diff
@@ -0,0 +1,37 @@
+---
+    title: 0007 Commit convention
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+In software development, maintaining clear and consistent commit message conventions is crucial for effective
+collaboration, code review, and project management. Commit messages serve as a form of documentation, helping developers
+understand the changes introduced by each commit without having to analyze the code diff extensively.
+
+## Decision
+
+A commit message must follow the following rules:
+
+1. The subject line (first line) MUST not be no longer than 50 characters
+2. The subject line MUST be in the imperative mood
+3. A sentences MUST have Capitalized first word
+4. The subject line MUST  not end with a punctuation
+5. The body SHOULD be restricted to 72 characters
+6. The body MUST be separate by a blank line from the subject line if used
+7. The body SHOULD be used to explain what and why, not how.
+8. The body COULD end with a ticket number
+9. The Subject line COULD include a ticket number in the following format
+
+  > \<ref\>-\<ticketnumber\>: subject line
+
```

Comment:
could you add an example of a valid commit message?

Reply:
done